components/dropdown/src/styles/style.scss

```diff
@@ -16,6 +16,13 @@
   position: relative;
 }
 
+#bibSizer {
+  position: absolute;
+  z-index: -1;
```

Comment:
Do we need this? It's not an element that you can focus, right?

Reply:
In fact... do we need  the pointer events rule either?

Reply:
`z-index:-1` will help the performance by putting the layer to be the most back so that other layer would get any cursor event. `pointer-events` rule would also help the performance with bypassing the cursor event on this layer. 
IMO, I think adding these rule wont hurt and rather help a little bit.